### PR TITLE
[Tech Glossary] OpenStack related terms

### LINKED ISSUE #1309: [RA-1 Terminology] Create Terminology Chapter
Relabel RA-1 Abbreviations to Terminology and capture content (terms and abbreviations).

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #1309

### CODE REVIEW

File: doc/tech/glossary.md [hunk 1]
```diff
@@ -106,8 +107,33 @@ Terms not defined by Kubernetes:
...
+>_*Note:*_ The official [OpenStack Glossary]( https://docs.openstack.org/image-guide/common/glossary.html) is an extensive list of OpenStack-related concepts. Some additional terms used in the Reference Architecture RA-1 or used to relate RA-1 terms with terms defined elsewhere.
+
+**Core (physical)** is an independent computer processing unit (pcpu) that can independently execute CPU instructions but is integrated with other cores on a multiprocessor chip (integrated circuit die). Please note that the multiprocessor chip is also referred to as a CPU that is placed in a socket of a computer motherboard.
+
+**Flavor Capabilities** such as (CPU Pinning, NUMA< huge pages): CNTT NFVI Profile
+
+**Flavor Geometry** Flavor sizing such as number of vCPUs, RAM, disk, etc.
+
+**Fluentd** is an open-source data collector that both collects data, transforms it and delivers it for consumption.
+
```
Comment (ASawwaf) [inline doc/tech/glossary.md]: **Fluentd ** is an open-source data collector for unified logging layer, which allows you to unify data collection and consumption for better use and understanding of data, it is CNCF graduated project "https://www.fluentd.org/"

File: doc/tech/glossary.md [hunk 2]
```diff
@@ -106,8 +107,33 @@ Terms not defined by Kubernetes:
 - **CaaS**:	Container-as-a-Service. A complete set of technologies to enable the management of containerised software, including a Kubernetes cluster, container networking, storage, routing, service mesh, etc.
 - **CaaS Manager**:	A management plane function that manages the lifecycle (instantiation, scaling, healing, etc.) of one or more CaaS instances, including communication with VIM for master/node lifecycle management.
 
-<a name="1.5"></a>
-## 1.5 Other Referenced Terminology
+
+<a name="1.6"></a>
```
Comment (msidana) [inline doc/tech/glossary.md]: ```<a name="1.6"></a>``` -> ```<a name="1.5"></a>```

File: doc/tech/glossary.md [hunk 3]
```diff
@@ -106,8 +107,33 @@ Terms not defined by Kubernetes:
...
+## 1.5 OpenStack Related Terminology
+
+>_*Note:*_ The official [OpenStack Glossary]( https://docs.openstack.org/image-guide/common/glossary.html) is an extensive list of OpenStack-related concepts. Some additional terms used in the Reference Architecture RA-1 or used to relate RA-1 terms with terms defined elsewhere.
+
+**Core (physical)** is an independent computer processing unit (pcpu) that can independently execute CPU instructions but is integrated with other cores on a multiprocessor chip (integrated circuit die). Please note that the multiprocessor chip is also referred to as a CPU that is placed in a socket of a computer motherboard.
+
+**Flavor Capabilities** such as (CPU Pinning, NUMA< huge pages): CNTT NFVI Profile
```
Comment (msidana) [inline doc/tech/glossary.md]: NUMA< huge pages -> NUMA, Hugepages

File: doc/tech/glossary.md [hunk 4]
```diff
@@ -106,8 +107,33 @@ Terms not defined by Kubernetes:
...
+
+**Flavor Geometry** Flavor sizing such as number of vCPUs, RAM, disk, etc.
+
+**Fluentd** is an open-source data collector that both collects data, transforms it and delivers it for consumption.
+
+**Host profile** is a set of underlay configuration values used to install a new compute host/server. Typically, the host profile configurations match the capabilities of one or more flavors.
+
+**Hugepages:** Physical memory is partitioned and accessed using the basic page unit (in Linux default size of 4 KB). Hugepages, typically 2 MB and 1GB size, allows large amounts of memory to be utilised with reduced overhead. In an NFV environment, huge pages are critical to support large memory pool allocation for data packet buffers. This results in fewer Translation Lookaside Buffers (TLB) lookups, which reduces the virtual to physical pages address translations. Without huge pages enabled high TLB miss rates would occur thereby slowing performance.
```
Comment (msidana) [inline doc/tech/glossary.md]: nit: slowing -> degrading

File: doc/tech/glossary.md (hunk 1 above)
Comment (pgoyal01) [reply doc/tech/glossary.md]: @ASawwaf Thanks. Changed to:
**[Fluentd](https://www.fluentd.org/)**, a CNCF graduated project, is an open-source data collector for unified logging layer, which allows data collection and consumption for better use and understanding of data.

File: doc/tech/glossary.md (hunk 2 above)
Comment (pgoyal01) [reply doc/tech/glossary.md]: @msidana Corrected -- Thanks

File: doc/tech/glossary.md (hunk 3 above)
Comment (pgoyal01) [reply doc/tech/glossary.md]: @msidana Corrected -- Thanks

File: doc/tech/glossary.md (hunk 4 above)
Comment (pgoyal01) [reply doc/tech/glossary.md]: @msidana Corrected -- Thanks

**Pull request merged**